src/i18n/languages/zh_CN.ts

```diff
@@ -35,4 +35,6 @@ export const zh_CN: Translation = {
 	[Key.author]: "作者",
 	[Key.publishedAt]: "发布于",
 	[Key.license]: "许可协议",
+
+	[Key.footnotes]: "脚注",
```

Comment:
根據臺灣的使用習慣，應該翻譯為「註腳」。
```suggestion
	[Key.footnotes]: "註腳",
```